pyramid_zipkin/tween.py

```diff
@@ -192,23 +192,33 @@ def tween(request: Request) -> Response:
             tween_kwargs['firehose_handler'] = zipkin_settings.firehose_handler
 
         with tracer.zipkin_span(**tween_kwargs) as zipkin_context:
-            response = handler(request)
-            if zipkin_settings.use_pattern_as_span_name and request.matched_route:
-                zipkin_context.override_span_name('{} {}'.format(
-                    request.method,
-                    request.matched_route.pattern,
-                ))
-            zipkin_context.update_binary_annotations(
-                get_binary_annotations(request, response),
-            )
-
-            if zipkin_settings.post_handler_hook:
-                zipkin_settings.post_handler_hook(
-                    request,
-                    response,
-                    zipkin_context
+            try:
+                response = None
```

Comment:
Do we need this line? The response variable is set again on the next line.

Reply:
In case of an exception the response variable won't get defined. And its used in the finally block. I moved outside of the try block.